clients/drcachesim/simulator/cache_simulator.h

```diff
@@ -60,12 +62,19 @@ class cache_simulator_t : public simulator_t {
     bool
     print_results() override;
 
+    int_least64_t
+    get_cache_metric(unsigned level, metric_name_t metric, unsigned core = 0,
```

Comment:
I would re-order the parameters with the metric first and grouping the 3 identifiers of which cache (level, core, split) together.